charts/ibm-odm-metering/RELEASENOTES.md

```diff
@@ -1,12 +1,12 @@
-# What's new in Helm chart 23.0.1
-The version 23.0.1 of the Helm chart installs version 8.12.0.0 of the IBM Operational Decision Manager metering service.
+# What's new in Helm chart 24.0.0
+The version 24.0.0 of the Helm chart installs version 9.0.0 of the IBM Operational Decision Manager metering service.
```

Comment:
Question: .env mentions ODMVERSION as 9.0.0.0 and the previous sentence mentioned 8.12.0.0 so it would be great to be in sync and decide between 3 and 4 digits version numbers.